### DISCUSSION TITLE
I need help on App created by Mr. Conorweb.

### DISCUSSION THREAD

**Discussion by sghriyadh** [Budibase/budibase#10779] [Help] [answered]
Hi 
Mr Conor @ConorWebb96 was very kind to create a small app for me recently. 
I tried to modify it but i cannot make one thing happen if you see the app.

I want user to "view" the row not in "side panel" or modal" . I want user to see it on a seperate screen .
I reckon it has something to do with "row id" and connecting it to a new screen. 

i am attaching the app here. if anyone can make this change and reupload it for me please?

File link : https://www.sendgb.com/upload/?utm_source=cHJkVb06vda

![ssssss](https://github.com/Budibase/budibase/assets/130876667/05010d56-b201-44a7-aa6b-99afd0e30c5c)

Thank you

**Comment by seoulaja** [comment]
Hi @sghriyadh 

Please refer to this documentation https://docs.budibase.com/docs/url-parameters

**Reply by sghriyadh** [reply]
Thank you. 

I did try various variable settings but i couldnot connect the "view" button to new screen showing full row record.

i am not that technical 

Will it be possible for you or anyone to have a look into the app and connect it to me. The zip is still attached.

That will be very kind of you.

**Reply by seoulaja** [reply]
Hi @sghriyadh 

I couldn't see the attached file, when I click the link it goes to sendgb website to upload file.

Make sure the destination page has a variable(s) setup. Below is my sample

/currency/
/currency/:id
 
The /currency is the page to list all the data, inside the table there is a link. Inside the view link on URL field I put this 
`/currency/{{ Currency Table.Currency._id }}` 

It means when the user clicks it, it will navigate to page currency with id: _id. 

I hope this helps.

Regards,
Seoul

**Reply by sghriyadh** [reply]
![image](https://github.com/Budibase/budibase/assets/130876667/a19fabf2-50da-472b-9209-e772da51258e)


i just checked it is there. i tried variables but it needs settings which are confusing to me as a non coders. Kindly help.

**Reply by seoulaja** [reply]
Hi @sghriyadh 

When I click https://www.sendgb.com/upload/?utm_source=cHJkVb06vda it sends me to below screen

![image](https://github.com/Budibase/budibase/assets/15101654/bf8e2973-d974-4bed-a4ae-241b3b7027e9)

The link doesn't work for me, can you use google drive or else?

**Reply by sghriyadh** [reply]
Here is the new link on a different website. 

https://file.io/DY0adDdbH0Le

I reached till creating /home/:id but it shows all records not the clicked one. . Can you reivew and fix it and send fixed Zip using same website please.

**Reply by seoulaja** [reply]
Hi @sghriyadh 

I cannot download the file, it says the file is deleted. Please make sure in your data provider on destination page you put a filter based on the url parameter, in this case :id 
![Screenshot 2023-06-07 at 01 17 11](https://github.com/Budibase/budibase/assets/15101654/8527103f-7f81-4b6b-9d47-557cdfb95f8b)

**Reply by sghriyadh** [reply]
Sorry . i think the link expired very quickly.

here is a valid link.https://www.transfernow.net/dl/20230606ZYKN21ku

I shall try your suggestion as well.

 kindly review this small app. if you can figure out whats missing. Thank you

**Reply by seoulaja** [reply]
Hi @sghriyadh 

I get this message when click the link 
![Screenshot 2023-06-07 at 06 11 47](https://github.com/Budibase/budibase/assets/15101654/d60c2b0f-119a-425d-94cd-09a025678cc6)

**Reply by sghriyadh** [reply]
I tried the filter but as soon as i apply the filter it hides all record. 
I found out that website are deleting gzip file uploaded so i now i uploaded it to a airtable.

https://airtable.com/shrZoAGgpXKMsdi4v

Can you download attachement from the link above and fix it please .

![image](https://github.com/Budibase/budibase/assets/130876667/5e6988df-0f20-4d93-909d-4ef144aa0d41)


 I am almost there but the filter is having some variable problems it seem.

You help will be grateful.

**Reply by seoulaja** [reply]
Hi @sghriyadh 

I experienced the same thing as what you got. The filter somehow doesn't work. So what I did was just re-apply the filter in data provider now it is working. 

Here is the updated one

[sghriyadh-export-1686293349736.tar.gz](https://github.com/Budibase/budibase/files/11703124/sghriyadh-export-1686293349736.tar.gz)

**Reply by sghriyadh** [reply]
Thank you. It is working. i turned off pagination and made limit to maximum 1 record per page.

**Reply by sghriyadh** [reply]
> Hi @sghriyadh
> 
> I experienced the same thing as what you got. The filter somehow doesn't work. So what I did was just re-apply the filter in data provider now it is working.
> 
> Here is the updated one
> 
> [sghriyadh-export-1686293349736.tar.gz](https://github.com/Budibase/budibase/files/11703124/sghriyadh-export-1686293349736.tar.gz)

I did notice one thing though.

It is saving the _id  in the Tile column instead of the procedure name( procedure names were UGI endoscopy , Lower GI endoscopy )

How can i fix this ?

![image](https://github.com/Budibase/budibase/assets/130876667/89cf4da6-fb83-4e0e-8834-4d137c5e47c7)

**Reply by seoulaja** [reply, marked as answer]
Hi @sghriyadh 

On Save button please map the field to be like below

![Screenshot 2023-06-10 at 17 17 34](https://github.com/Budibase/budibase/assets/15101654/cdc59e1d-99b8-422d-b7df-d25ba2fd9033)

It will tell Budibase what component's value belongs to what column
